examples/complete/main.tf

```diff
@@ -117,10 +117,6 @@ module "ecs_alb_service_task" {
           port     = 80
           }
         ]
-        timeout = [{
```

Comment:
@awkaplan FYI, I couldn't get this working with a TCP port.

When Timeout is supplied, even if `per_request_timeout_seconds` is not supplied, I think the provider is setting the value to 0 which is invalid for a TCP port.

I haven't looked into it in detail, but it would be good if we could fix this as it will come up again.